department-of-veterans-affairs/va.gov-cms
# VAGOV-6343: Removing field_media from detail pages.

## What this does
 - Removes deprecated `field_media` from `health_care_region_detail_page` content type

## QA
 - Visit `/admin/structure/types/manage/health_care_region_detail_page/fields` and visually verify that `field_media` does not appear.
